modules/xfeatures2d/samples/shape_transformation.cpp

```diff
@@ -2,10 +2,10 @@
  * shape_context.cpp -- Shape context demo for shape matching
  */
 #include <iostream>
+#include "opencv2/shape.hpp"
```

Comment:
Use `#include "opencv2/opencv_modules.hpp"` for proper `HAVE_OPENCV_SHAPE` definition.